README.md

```diff
@@ -9,3 +10,12 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+
+Bijectivity means that each unique node in graph $A$ can be mapped to a corresponding unique node in graph $B$. This means that there must be the same number of nodes in both graphs. <br><br>
+A graph is completely connected when every pair of distinct vertices is connected by a unique edge. <br><br>
+Consider the edge between distinct nodes $u$ and $v$ in graph $A$. Because $A$ is completely connected, there must be an edge between them. And since both graphs are bijective, $(f(u), f(v))$ must exist in graph $B$. This goes for every vertex in both graphs.<br><br>
```

Comment:
What is $f$?

Reply:
f is the bijective function that maps each node in graph A
 to each node in graph B. I updated the description to be more clear about this